dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.mpMetrics-5.0-monitor-1.0.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpMetrics-5.0-monitor-1.0
+visibility=private
+IBM-Provision-Capability: \
```

Comment:
mpMetrics-5.0 depends on monitor-1.0 so this auto feature is not needed and the bundle should be moved into the mpMetrics-5.0 feature file?